libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/ExternalPlatformDropdown.tsx

```diff
@@ -193,15 +181,15 @@ export const ExternalPlatformDropdown = ({
             <Tooltip hidden={!disabledReason} content={disabledReason} position="top">
               <div>
                 {label}
-                {isPlatformDevPreview(
-                  platform as PlatformType,
-                  newFeatureSupportLevelContext,
-                  featureSupportLevelData,
-                ) && (
-                  <span onClick={(event) => event.stopPropagation()}>
-                    <DeveloperPreview testId={'platform-support-level'} />
-                  </span>
-                )}
+                <span onClick={(event) => event.stopPropagation()}>
+                  <NewFeatureSupportLevelBadge
+                    featureId={ExternalPlaformIds[platform as PlatformType] as FeatureId}
+                    supportLevel={newFeatureSupportLevelContext.getFeatureSupportLevel(
+                      ExternalPlaformIds[platform as PlatformType] as FeatureId,
+                      featureSupportLevelData ?? undefined,
+                    )}
```

Comment:
Why not have this as part of the `getExternalPlatformTypes` function as a new `supportLevel` field? So it's all in one place?

Reply:
Good idea. Changed